addons/html_editor/static/src/main/toolbar/toolbar.xml

```diff
@@ -16,7 +16,8 @@
                                 {{ state.buttonsDisabledState[button.id] ? 'disabled': ''}}"
                             t-att-title="button.name"
                             t-att-name="button.id"
-                            t-on-click="() => button.action(this.props.toolbar.dispatch)">
+                            t-on-click="() => button.action(this.props.toolbar.dispatch)"
+                            t-att-hidden="state.buttonsHiddenState[button.id]">
```

Comment:
Like in the powerbox, we want a way to specify isAvailable base on the selection. And we want to use t-if to display or not the button